test/mk/components.mk

```diff
@@ -2,7 +2,10 @@
 
 FIPS202_SRCS = $(wildcard mldsa/fips202/*.c)
 SOURCES += $(wildcard mldsa/*.c)
-
+ifeq ($(OPT),1)
+	SOURCES += $(wildcard mldsa/native/aarch64/src/*.[csS]) $(wildcard mldsa/native/x86_64/src/*.[csS])
+	CFLAGS += -DMLD_CONFIG_USE_NATIVE_BACKEND_ARITH -DMLD_CONFIG_USE_NATIVE_BACKEND_FIPS202
```

Comment:
Nit: We don't have `MLD_CONFIG_USE_NATIVE_BACKEND_FIPS202` yet, but we will soon, so we can keep it.